chromium_src/chrome/install_static/chromium_install_modes.cc

```diff
@@ -25,7 +25,7 @@ const size_t kProductPathNameLength = _countof(kProductPathName) - 1;
 
 const char kSafeBrowsingName[] = "chromium";
 
-#if defined(OFFICIAL_BUILD)
+#if !defined(COMPONENT_BUILD)
```

Comment:
one more in `brave/chromium_src/chrome/install_static/chromium_install_modes.h`

Reply:
We can't simple change chromium_install_modes, it's lead to the chain reactions in `brave/chromium_src/chrome/install_static/*`
This is only a test change, we shouldn't commit it.

Reply:
Reverted.